deploy/helm/sumologic/templates/_helpers/_common.tpl

```diff
@@ -149,6 +151,7 @@ machineconfiguration.openshift.io/role: worker
 
 {{- define "sumologic.labels.app.machineconfig.master" -}}
 {{- template "sumologic.fullname" . }}-master-extensions
+sumologic.com/app: otelcol-events
```

Comment:
not sure if this is the right place?

Reply:
no it's not. You should create new template analogically, I would go with `sumologic.labels.app.events` name